data_validation/query_builder/query_builder.py

```diff
@@ -279,7 +301,7 @@ def hash(config, fields):
     def ifnull(config, fields):
```

Comment:
confusing to read.

1. is default_string used somewhere downstream?
2. why is it valid to take an arbitrary first value in fields and then overwrite itself?  Is this really the same object?

Reply:
agreed it's confusing. The default string is the replacement value when calling the ibis IFNULL equivalent. Looking at the statement it never actually reads from the config so I'll get that fixed.

https://ibis-project.org/docs/generated/ibis.expr.api.ValueExpr.fillna.html

Disclaimer: the only reason this was updated in this PR is because the order of inputs was backward.